pkg/discovery/worker/group_discovery_worker.go

```diff
@@ -155,6 +157,32 @@ func (worker *k8sEntityGroupDiscoveryWorker) buildSidecarContainerSpecGroup(side
 	return groupsDTOs
 }
 
+func (worker *k8sEntityGroupDiscoveryWorker) buildPodsWithVolumesGroup(podsWithVolumes []string) []*proto.GroupDTO {
+	var groupsDTOs []*proto.GroupDTO
+	if len(podsWithVolumes) <= 0 {
+		return groupsDTOs
+	}
+	id := fmt.Sprintf("All-Pods-Using-Volumes-%s", worker.targetId)
+	displayName := "All Pods Using Volumes"
```

Comment:
Should the display name be dynamically generated like the ID is here? It seems like this could result in a scenario where there are multiple groups generated, all with the same display name, but with differing contents.

Reply:
@amd-ibm there's a special server side magic( :) ) that differentiates the gourps with same name but coming from different clusters.
It identifies the groups uniquely by `id` field and attaches the cluster identifier with each discovered group and can differentiate the same in the UI, eg. you would see the cluster name displayed below the group:
<img width="288" alt="image" src="https://user-images.githubusercontent.com/10027921/184287479-47e0860e-aae0-43bd-ae14-af154a759c74.png">
We have many such auto discovered groups with static common names across different clusters. For example:
<img width="288" alt="image" src="https://user-images.githubusercontent.com/10027921/184287588-fe15ce18-5dab-4131-9e6f-d7b7e53e39a4.png">

Reply:
I was unaware of that. Thanks for the info.